settings.gradle.kts

```diff
@@ -1,3 +1,9 @@
+pluginManagement {
+    repositories {
+        gradlePluginPortal()
+        google()
+    }
+}
```

Comment:
I had to add this so that we could use the `com.android.library` plugin in the root `build.gradle`.